src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,64 @@
  */
 
 public class Plane {
+    private final String brand;
+    private final String model;
+    private final int passengersCapacity;
+    private final int crewCount;
+    private final int takeOffWeightTon;
 
+    private Plane(PlaneBuilder planeBuilder) {
+        this.brand = planeBuilder.brand;
```

Comment:
```suggestion
        brand = planeBuilder.brand;
```
те ж саме для решти полів